src/components/label-pill/LabelPill.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import classNames from 'classnames';
+
+import './LabelPill.scss';
+
+export enum LabelPillStatus {
+    DEFAULT = 'default',
+    INFO = 'info',
+    HIGHLIGHT = 'highlight',
+    SUCCESS = 'success',
+    WARNING = 'warning',
+    ALERT = 'alert',
+    ERROR = 'error',
+}
+
+export interface LabelPillProps {
+    children: React.ReactNode;
+    type?: LabelPillStatus;
+}
+
+const LabelPill = React.forwardRef((props: LabelPillProps, ref: React.Ref<HTMLSpanElement>) => {
```

Comment:
Should we now call it just "Pill" hoping its one Pill to rule them all

Reply:
I went with "labelPill" in this case to avoid a conflict with the PillCloud component (which already uses the namespace "bdl-Pill" unfortunately).

Reply:
oh yeah, and there's a Pill inside PillSelector dropdown. the styles are shared between these two..